packages/react/src/contexts/CorbadoSessionContext.tsx

```diff
@@ -13,6 +13,7 @@ export interface CorbadoSessionContextProps {
   loading: boolean;
   isAuthenticated: boolean;
   user: SessionUser | undefined;
+  skipHashedUrls: boolean;
```

Comment:
Why is this part of the react session context? To me this is a setting like any other setting (e.g. projectId or setShortSessionCookie) => it should be in CorbadoAppParams.

Or am I missing something?

Reply:
@Aby-JS I tried to use the setting but it's not available to me in CorbadoAuth and the other components => Did you test this and if yes how can I use the setting?

Reply:
apologies for not testing and missing this. Fixed and tested. You can test now